packages/framework/src/Controller/Admin/ProductController.php

```diff
@@ -167,7 +168,12 @@ public function editAction(Request $request, $id)
      */
     public function newAction(Request $request)
     {
-        $productData = $this->productDataFactory->create();
+        try {
+            $productData = $this->productDataFactory->create();
+        } catch (HttpException $e) {
```

Comment:
We (me+David) decided to catch general Http Not Found Exception because it is used in framework for not founds. When user extends the factory and add a new default field (and if have a clean database), this will catch it and redirect to dashboard where should be warnings about not-setted default values.

Reply:
OK, that makes sense, so how about catching a bit more specific `\Symfony\Component\HttpKernel\Exception\NotFoundHttpException`?

Reply:
Yep, it was a mistake, chaning...